packages/schema/package.json

```diff
@@ -26,7 +26,11 @@
 	"author": "Rijk van Zanten <rijkvanzanten@me.com>",
 	"exports": {
 		".": "./dist/index.js",
-		"./package.json": "./package.json"
+		"./package.json": "./package.json",
+		"./dist/types/overview": {
```

Comment:
```suggestion
		"./types/overview": {
```

Import in `src/utils/get-default-value.ts` needs to be adjusted:
```diff
- import { SchemaOverview } from '@directus/schema/dist/types/overview';
+ import { SchemaOverview } from '@directus/schema/types/overview';
```

Tested locally, it works ✅